### ISSUE TITLE
many deprecation errors with Qt 5.15.1

### ISSUE THREAD

**Issue by Geremia** [openstreetmap/merkaartor#220]
I get many deprecation build errors like these (Merkaartor 0.18.4):
```Dans le fichier inclus depuis /usr/include/qt5/QtGui/qtguiglobal.h:43,
                 depuis /usr/include/qt5/QtGui/qpixmap.h:43,
                 depuis /usr/include/qt5/QtGui/QPixmap:1,
                 depuis ../interfaces/IImageManager.h:23,
                 depuis ../interfaces/IImageManager.cpp:20:
/usr/include/qt5/QtCore/qglobal.h:1279:80: note: déclaré ici
 1279 | Q_CORE_EXPORT QT_DEPRECATED_VERSION_X_5_15("use QRandomGenerator instead") int qrand();
      |                                                                                ^~~~~
make[2]: *** [Makefile.Release:4640: release/ExtrudeInteraction.o] Error 1
make[2]: *** [Makefile.Release:4876: release/StyleDock.o] Error 1
make[2]: *** [Makefile.Release:4749: release/PropertiesDock.o] Error 1
make[2]: *** [Makefile.Release:4846: release/DirtyDock.o] Error 1
make[2]: *** [Makefile.Release:4922: release/FeaturesDock.o] Error 1
make[2]: *** [Makefile.Release:4807: release/LayerDock.o] Error 1```

**Comment by zander** [comment]
I believe this should be fixed in fc68405f and 5df96276

**Comment by Krakonos** [comment]
Tested on Arch with Qt 5.15.2. Most of the warnings are gone thanks to the patches from @zander . There are some GDAL related warnings not related to Qt.
